src/main/java/uk/gov/digital/ho/hocs/domain/exceptions/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.digital.ho.hocs.domain.exceptions;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static org.springframework.http.HttpStatus.SERVICE_UNAVAILABLE;
+import static uk.gov.digital.ho.hocs.application.LogEvent.EVENT;
+
+@ControllerAdvice
+@Slf4j
+public class RestResponseEntityExceptionHandler {
+
+    @ExceptionHandler(ApplicationExceptions.TooManyMessagesException.class)
+    public ResponseEntity<String> handle(ApplicationExceptions.TooManyMessagesException e) {
```

Comment:
This feels like a good opportunity, either or in the future, to design a proper error model.